build/setup.mk

```diff
@@ -4,8 +4,13 @@ ifeq ($(GO),)
     $(error "go is not available: see https://golang.org/doc/install")
 endif
 
+# Gather build variables to inject into the manifest tool
+BUILD_HASH_SHORT = $(shell git rev-parse --short HEAD)
+BUILD_TAG_LATEST = $(shell git describe --tags --match 'v*' --abbrev=0)
```

Comment:
Any way we can avoid logging the error if no tag was found? (`No names found, cannot describe anything.`)

Reply:
Indeed, fixed!